src/transformers/models/ijepa/modeling_ijepa.py

```diff
@@ -189,6 +188,36 @@ def _init_weights(self, module: Union[nn.Linear, nn.Conv2d, nn.LayerNorm]) -> No
             ).to(module.position_embeddings.dtype)
 
 
+def eager_attention_forward(
```

Comment:
missing copied from

Reply:
That's the original one, other models are copied from it